package.json

```diff
@@ -43,6 +42,8 @@
     "@slack/events-api": "1.0.1",
     "chai": "4.1.0",
     "mocha": "3.4.2",
+    "nock": "9.0.14",
+    "proxyquire": "^1.8.0",
```

Comment:
please don't use ^

Reply:
Removed. And I misspelled json in the commit message...